features/0434-outofband/README.md

```diff
@@ -360,13 +362,13 @@ https://<domain>/<path>?oob=<outofbandMessage>
 
 > To do: We need to rationalize this approach `https://` approach with the use of a special protocol (e.g. `didcomm://`) that will enable handling of the URL on mobile devices to automatically invoke an installed app on both Android and iOS. A user must be able to process the out-of-band message on the device of the agent (e.g. when the mobile device can't scan the QR code because it is on a web page on device).
 
-The `<outofbandMessage>` is an agent plaintext message (not a DIDComm message) that has been base64 url encoded.
+The `<outofbandMessage>` is an agent plaintext message (not a DIDComm message) that has been Base64Url encoded and further Uri (percent) encoded for special characters that may be generated in the base64 string.
```

Comment:
The goal of Base64URL encoding is specifically to avoid characters ( '+', '/' and '=') that need percent-encoding. Why the extra step here, exactly?

Reply:
I think it's meant as a clarification about what Base64URL encoding it vs. a second step.  Maybe:

` Base64URL encoded such that the resulting string can be safely used in a URL.`